Projects/hCore/Sources/UIView+AllowTouchesOfViewsOutsideBounds.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  UIView+AllowTouchesOfViewsOutsideBounds.swift
+//  hCore
+//
+//  Created by Sam Pettersson on 2021-04-20.
+//  Copyright © 2021 Hedvig AB. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public extension UIView {
+    private struct ExtendedTouchAssociatedKey {
+        static var outsideOfBounds = "viewExtensionAllowTouchesOutsideOfBounds"
+    }
+
+    /// This propery is set on the parent of the view that first clips the content you want to be touchable
+    /// outside of the bounds
+    var allowTouchesOfViewsOutsideBounds:Bool {
+        get {
+            return objc_getAssociatedObject(self, &ExtendedTouchAssociatedKey.outsideOfBounds) as? Bool ?? false
```

Comment:
Can this ever return true?

Reply:
Ahhh I see, it's like it has an initial value of false. This is kind of hidden though. How can you set a `Bool` to a `String` parameter!? Does it create a pointer?